src/Raven.Server/NotificationCenter/Notifications/AlertType.cs

```diff
@@ -83,6 +83,8 @@ public enum AlertType
         MismatchedReferenceLoad,
 
         BlockingTombstones,
-        ServerLimits
+        ServerLimits,
+
+		Revisions
```

Comment:
Isn't to generic name?

Reply:
Yes, that is exactly @ml054 point. This name is too generic, we already have `RevisionsConfigurationNotValid` and all other names are more specific.

Reply:
Done